info.json

```diff
@@ -14,43 +14,27 @@
 	"srcBuildFolderName": "event-espresso-core",
 	"releaseFilesRemove": [
 		"src\/tests\/**",
-		"src\/acceptance_tests\/**",
```

Comment:
out of curiosity - where is this `info.json` being used?

Reply:
`info.json` is used by the [ee build machine](https://github.com/eventespresso/grunt-wp-plugin-buildmachine) during deployment / release